### PR TITLE
Fix python dependency

### PR DESCRIPTION
**Pull request by ocefpaf**
Python 3.5! (For real now :smile:)

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,7 +16,6 @@ build:
 requirements:
     build:
         - python
-        - python >=2.7,<3.5  # See https://github.com/SciTools/cartopy/pull/676
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Once I removed these lines the `rerender` took care of the changes above.

**Comment by pelson** [comment]
Sorry. I completely missed that too! :cry:

**Comment by janschulz** [comment]
Looks like the builder is stuck :-(

**Comment by ocefpaf** [comment]
Not really. staged-recipes takes precedence from the feedstocks, so we have to wait...

**Comment by janschulz** [comment]
Huih, a whole day? That's bad... :-/

**Comment by ocefpaf** [comment]
I've been waiting for a few packages for days! It took ~5 days for `gdal`'s chance in AppVeyor :wink:

**Comment by pelson** [comment]
> Not really. staged-recipes takes precedence from the feedstocks, so we have to wait...

You know, I don't that has been configured by me. Maybe it's queue is checked first or something.

**Comment by ocefpaf** [comment]
I don't know why that happens either, but AppVeyor stops a feedstock project matrix to finish a staged-recipes PR. That is why I believe de-coupling those two will help us a lot to get the files uploaded in time.

**Pull request merged**